fswatcher/fswatcher.go

```diff
@@ -63,36 +73,143 @@ func (w *Sentry) Watch() (certCh <-chan tls.Certificate, errCh <-chan error) {
 		err := w.fsnotify.Add(w.certPath)
 
 		if err != nil {
-			w.errChan <- err
+			select {
```

Comment:
seems like this select is not needed with the one below it?

Reply:
If the channel is closed, we want to return.